recipes/pvxs/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cat <<EOF > configure/RELEASE.local
+EPICS_BASE=${EPICS_BASE}
+EOF
+
+make install
+
+install -d ${PREFIX}/include/pvxs ${PREFIX}/bin/ ${PREFIX}/lib/ ${PREFIX}/epics/modules/pvxs/dbd
```

Comment:
Actually, one other point here. Why are we installing the .dbd files into epics/modules/pvxs/dbd and not just pvxs/dbd?